src/main/kotlin/org/jitsi/jibri/selenium/status_checks/EmptyCallStatusCheck.kt

```diff
@@ -44,6 +46,9 @@ class EmptyCallStatusCheck(
     private fun CallPage.isCallEmpty() = getNumParticipants() <= 1
 
     companion object {
-        val DEFAULT_CALL_EMPTY_TIMEOUT: Duration = Duration.ofSeconds(30)
+        private val defaultCallEmptyTimeout: Duration by config {
+            "jibri.call-status-checks.default-call-empty-timeout".from(Config.configSource)
+        }
+        val DEFAULT_CALL_EMPTY_TIMEOUT: Duration = defaultCallEmptyTimeout
```

Comment:
I don't think we need the separate value here, let's get rid of `DEFAULT_CALL_EMPTY_TIMEOUT` and just remove the private from `defaultCallEmptyTimeout`